mobx/lib/home_page.dart

```diff
@@ -4,105 +4,122 @@ import 'package:mobx/mobx.dart';
 import 'package:mobx_sample/add_todo_page.dart';
 import 'package:mobx_sample/model/todo.dart';
 import 'package:mobx_sample/model/todo_list.dart';
+import 'package:mobx_sample/stats_counter.dart';
 import 'package:mobx_sample/todo_details_page.dart';
 import 'package:provider/provider.dart';
 
-class HomePage extends StatelessWidget {
+class HomePage extends StatefulWidget {
```

Comment:
Why does this have to be a StatefulWidget ? Can we move the state out into the store and control which tab is visible at a higher level ?